open-budget/app/styles/main.less

```diff
@@ -2242,29 +2886,155 @@ svg rect{
   // overflow: auto;
 }
 
-  .node {
-    border: solid 1px white;
-    font: 10px sans-serif;
-    line-height: 12px;
-    overflow: hidden;
-    position: absolute;
-    text-indent: 2px;
-  }
+.node {
+  border: solid 1px white;
+  font: 10px sans-serif;
+  line-height: 12px;
+  overflow: hidden;
+  position: absolute;
+  text-indent: 2px;
+}
 
 #bubble-chart-container {
   position: relative;
 }
-  .bubble-group-label {
-    position: absolute;
+
+.bubble-group-label {
+  position: absolute;
+  text-align: center;
+  pointer-events: none; // in order to allow hover event to fire under the label
+}
+
+.bubble-group-label-child {
+  position: relative;
+  left: -50%;
+}
+
+.bubble-group-label-total-sum {
+  font-weight: bold;
+}
+
+// Exemptions
```

Comment:
Are the only changes you've made after this label? 
Or did you make any other changes above it?